easybuild/easyconfigs/l/libosmium/libosmium-2.15.4-foss-2018b-Python-3.6.6.eb

```
@@ -0,0 +1,47 @@
+easyblock = "CMakeMake"
+
+name = 'libosmium'
+version = '2.15.4'
+versionsuffix = '-Python-3.6.6'
+
+homepage = "https://osmcode.org/libosmium/"
+description = """The Osmium Library has extensive support for all types of OSM entities:
+ nodes, ways, relations, and changesets.
+ It allows reading from and writing to OSM files in XML and PBF formats, including change files and full history files.
+ Osmium can store OSM data in memory and on disk in various formats and using various indexes.
+ Its easy to use handler interface allows you to quickly write data filtering and conversion functions.
+ Osmium can create WKT, WKB, OGR, GEOS and GeoJSON geometries for easy conversion into many GIS formats
+ and it can assemble multipolygons from ways and relations."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/osmcode/libosmium/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['402247f7764bc20c061fa7d9fad15fc85459b7ce4f420274de78c94d05ddf260']
+
+builddependencies = [
+    ('CMake', '3.12.1'),
+    ('Doxygen', '1.8.14'),
+    ('Graphviz', '2.40.1'),
+]
+
+dependencies = [
+    ('protozero', '1.6.8'),
+    ('expat', '2.2.5'),
+    ('zlib', '1.2.11'),
+    ('bzip2', '1.0.6'),
+    ('Boost', '1.67.0'),
+    ('GDAL', '2.2.3', versionsuffix),
+    ('GEOS', '3.6.2', versionsuffix),
+    ('PROJ', '5.0.0'),
+    ('sparsehash', '2.0.3'),
+]
+
+configopts = "-DINSTALL_UTFCPP=ON"
+
```

@tstrempel it would be good to run the tests as well here:
```suggestion

runtest = 'test'

```